### DISCUSSION TITLE
CP-SAT  miniSAT

### DISCUSSION THREAD

**Discussion by C8504** [google/or-tools#2922] [CP-SAT questions] [answered]
Hi Experts,

Could I ask a queastion that where I can get comparision between CP-SAT with miniSAT in terms of performance, features, etc?
thanks a lot.

John

**Comment by lperron** [comment, marked as answer]
We do not tune for pure sat problems. So maybe minisat is ahead. Now, CP-SAT is a CP solver, and is the best one in the world.

**Reply by C8504** [reply]
Agree!
Look forward to somebody run benchmark on pure sat problems.
